spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/metrics/GcpStackdriverMetricsAutoConfiguration.java

```diff
@@ -47,8 +50,9 @@
 @Configuration(proxyBeanMethods = false)
 @AutoConfigureBefore(StackdriverMetricsExportAutoConfiguration.class)
 @AutoConfigureAfter(MetricsAutoConfiguration.class)
-@ConditionalOnClass({StepMeterRegistry.class, StackdriverConfig.class})
+@ConditionalOnClass(StackdriverMeterRegistry.class)
 @ConditionalOnBean(Clock.class)
+@ConditionalOnEnabledMetricsExport("stackdriver")
 @EnableConfigurationProperties({GcpMetricsProperties.class, StackdriverProperties.class})
 @ConditionalOnProperty(value = "spring.cloud.gcp.metrics.enabled", matchIfMissing = true, havingValue = "true")
```

Comment:
Do we need both this *and* `@ConditionalOnEnabledMetricsExport`?

Reply:
They use different properties to disable the module. One way is through actuator and one through our standard `spring.cloud.gpc.metrics.enabled`.